.prow.yaml

```diff
@@ -40,6 +40,7 @@ presubmits:
     run_if_changed: "^(cmd|codegen|hack|pkg)/"
     decorate: true
     clone_uri: "ssh://git@github.com/kubermatic/kubermatic.git"
+    path_alias: k8c.io/kubermatic
```

Comment:
```suggestion
    path_alias: k8c.io/kubermatic/v2
```

Reply:
This would be wrong. The repo is not cloned to /v2. Or is it? I don't have it there, I have it in $GOPATH/src/k8c.io/kubermatic.

Reply:
I've tried this way (moved my local repo into $GOPATH/src/k8c.io/kubermatic/v2) and codegen was perfect!

Reply:
Forgive my language, but having the entire repo unter `v2/` is just .... so ugly that I would backpeddle on the whole proposal and not include the version in the module name at all :-D

Reply:
The point is that "codegen shell scripts" from k8s.io/code-generator don't care about modules and always treat package names as paths, thus — they will generate to the `$GOPATH/src/k8c.io/kubermatic/v2`. SO in order to have normally working codegen locally one will have to git clone into the `$GOPATH/src/k8c.io/kubermatic/v2`.

Reply:
Or, how about we keep the two lines in the bash script and thereby get the best of both worlds? Working codegen *and* a sane repository structure, where `../dashboard` is actually still the dashboard. Nobody will ever do a `git clone https://github.com/kubermatic/kubermatic v2`.

Reply:
I mean, sure, on CI nobody cares about the path_alias. But I think it's good to have the script work on CI and locally, so both should use the same filesystem location for the repository.

Reply:
I want to left a warning for the reference, because it caught me unguarded. With go mods it's now possible to place go sources **anywhere**, not just in `$GOPATH/src/<FDPN>`, but codegen doesn't know about this and it will `mkdir -p $GOPATH/src/k8c.io/kubermatic/v2` and write files there, no matter where your real kubermatic is in the FS.